### ISSUE TITLE
Continuous (loop) satin rails make string art

### ISSUE THREAD

**Issue by tyrosinase** [inkstitch/inkstitch#2700]
If satin rails are continuous loops, Ink/Stitch starts from an arbitrary point (from the user's perspective) on each rail. Even if there are rungs, if the two points are offset it makes "string art" that resembles a reversed-rail situation.

It should probably throw a warning (in Troubleshooting, at least) about this, if it can't just use rungs to choose a better starting point for both rails.

I broke and re-joined the two nodes that are no longer smooth, and the next node from them became first in the subpaths.

![image](https://github.com/inkstitch/inkstitch/assets/9473609/42a9a562-91c2-4744-a40c-a59e39c811ef)

By default the auto-rail reverses tried to "fix" it which... doesn't work.

![image](https://github.com/inkstitch/inkstitch/assets/9473609/de95aaae-0950-4daf-9064-8845e926707a)

Turning off auto-reverse gives exactly what one would expect: the runs chase each other around the circle.

![image](https://github.com/inkstitch/inkstitch/assets/9473609/88031d9c-4173-4a41-a12e-eef12fe1f9bd)

Introducing rungs clearly makes it try *something*, but it doesn't affect the starting points.

![image](https://github.com/inkstitch/inkstitch/assets/9473609/13d359cc-6faf-4e0d-a531-14a5b246c5ff)

**Comment by kaalleen** [comment]
It does exactly what you would suppose it to do. Using a rung as a breaking point (if present) is an interesting idea though.

**Comment by tyrosinase** [comment]
I don't think that's what anyone who isn't familiar with Ink/Stitch's workings would suppose it to do.

It caused a *lot* of confusion to a non-developer, and honestly I was pretty confused by it when I first saw it happen. It wasn't until they sent me the file and I realized there was an unbroken loop (actually one rail was a loop and one wasn't in that case) that I even understood what was happening.
